program/src/error.rs

```diff
@@ -163,9 +163,13 @@ pub enum LidoError {
     #[error("ValidatorShouldHaveNoStakeAccounts")]
     ValidatorShouldHaveNoStakeAccounts = 41,
 
+    /// Tried to remove a validator when it when it was active or had stake accounts.
+    #[error("ValidatorShouldHaveNoUnStakeAccounts")]
+    ValidatorShouldHaveNoUnStakeAccounts = 42,
+
     /// There is a validator that has less stake than the selected one, stake to that one instead.
     #[error("ValidatorWithLessStakeExists")]
-    ValidatorWithLessStakeExists = 42,
+    ValidatorWithLessStakeExists = 43,
```

Comment:
Hey, we shouldn’t renumber these error codes once we’ve defined them :sweat_smile: 

For now nothing bad will happen because this is not yet deployed, but we should make a habit of keeping them stable.

Reply:
😬 Reverted it.